simplegrid/dqn_agent.py

```diff
@@ -35,27 +35,28 @@ def remember(self, state, action, reward, next_state, done):
 
     # get action
     def act(self, state):
-
-        # select random action with prob=epsilon else action=maxQ
+        '''
+        state is a 2-dimensional ndarray representing the surrounding squares
+        returns a random action with prob=epsilon else action=maxQ
+        '''
```

Comment:
@hpharmsen - zie https://www.python.org/dev/peps/pep-0257/

In het algemeen wil je voor een doc string """ gebruiken en direct na de eerste een zinnetje hebben. Als je meer dan een zin hebt, een lege regel en de rest van het verhaal.

Reply:
Aangepast. Volgens  https://www.python.org/dev/peps/pep-0257/ geen lege regel na multiline docstring overigens. Alleen een regel met de closing quotes.